### PR TITLE
Forceios now lives in SalesforceMobileSDK-iOS

### PR DESCRIPTION
**Pull request by wmathurin**
The npm package generation is really quick (under 5s) and the generated package is much smaller (about 5.8 MB)

### CODE REVIEW

**Comment by wmathurin** [comment]
prepublish.js and postpublish.js are now identical to the SalesforceMobileSDK-Android's versions and will be moved to SalesforceMobileSDK-Shared in the next pull request

**Comment by wmathurin** [comment]
We no longer need the SalesforceMobileSDK-iOS-Package and the SalesforceMobileSDK-iOS-Distribution repos

**Comment by bhariharan** [comment]
We can remove the dependency on the ```Dependencies``` repo too (it only has ```libSqlcipher.a```), but that can be another pull request.

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+external/fmdb
+external/ocmock
+external/ThirdPartyDependencies
+external/shared/samples/accounteditor/
+external/shared/samples/contactexplorer/
+external/shared/samples/fileexplorer/
+external/shared/samples/simplesync/
+external/shared/samples/smartstoreexplorer/
+external/shared/samples/userandgroupsearch/
+external/shared/samples/usersearch/
```
Comment (bhariharan) [inline .npmignore]: Should we add ```xcuserdata``` and ```xcshareddata``` to this too? Maybe even the ```workspace``` file/folder?

File: node/forceios.js [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+        usage();
+        process.exit(4);
+    }
+
+    //
+    // FIXME
+    // 
+
+    /*
+    // Copy dependencies
```
Comment (bhariharan) [inline node/forceios.js]: Should this be removed?

**Comment by bhariharan** [comment]
Looks good otherwise.

File: node/forceios.js (hunk 2 above)
Comment (wmathurin) [inline node/forceios.js]: I think so - we don't have it on the android side.
We could fix it to do pod update or cordova update but maybe it's best they use pod / cordova directly

File: .npmignore (hunk 1 above)
Comment (wmathurin) [inline .npmignore]: We could

**Pull request merged**